File: ticketProject6-RMI/TicketSystem/TicketNetworking/src/main/java/network/objectprotocol/BuyTicketRequest.java
package network.objectprotocol;

import network.dto.ArtisticRepresentationDTO;

public class BuyTicketRequest implements Request{
    private ArtisticRepresentationDTO arDTO;

    public BuyTicketRequest(ArtisticRepresentationDTO arDTO) {
        this.arDTO = arDTO;
    }

    public ArtisticRepresentationDTO getArDTO() {
        return arDTO;
    }
}

File: ticketProject7-CrossPlatform/TS-crossp/networking/request/BuyTicketRequest.cs
using System;
namespace networking.protocol
{

	using ArtisticRepresentationDTO = networking.dto.ArtisticRepresentationDTO;

	[Serializable]
	public class BuyTicketRequest : Request
	{
		private ArtisticRepresentationDTO ar;

		public BuyTicketRequest(ArtisticRepresentationDTO ar)
		{
			this.ar = ar;
		}

		public virtual ArtisticRepresentationDTO ArtisticRepresentation
		{
			get
			{
				return ar;
			}
		}
	}

}